### ISSUE TITLE
Quest:9211 "The Ice Guard" Should be repeatable

### ISSUE THREAD

**Issue by FenrirNost** [elysium-project/server#2489]
The quest "The Ice Guard" (9211) should be repeatable.

Evidence from Nov 2007: https://web.archive.org/web/20071103001520/http://wow.allakhazam.com:80/db/quest.html?wquest=9211
See the top left, "Repeatable: Yes"

Evidence from June 2008:
http://db.vanillagaming.org/?quest=9211#wh-comments
See the first comment "This quest does not count towards the achievements total.", this is because it is a repeatable quest.

Also I think it is relevant to note that in searching for information on this quest I found nothing to indicate that it should NOT be repeatable. If anyone has additional evidence, please post it but all I could find was these two things that suggest it was indeed repeatable.

It was also added at the same time as the savage guard "New quests available in Zul'Gurub and Light's Hope Chapel for head and leg resistance enchantments." (1.11.0 patch notes). I think it would make sense that it works the same way as the savage guard quest.

All 3 savage guard quests are also listed as repeatable as of Oct 2007.
https://web.archive.org/web/20071028153928/http://wow.allakhazam.com:80/db/quest.html?wquest=9208
https://web.archive.org/web/20071028153934/http://wow.allakhazam.com:80/db/quest.html?wquest=9209
https://web.archive.org/web/20071028123038/http://wow.allakhazam.com:80/db/quest.html?wquest=9210

There is a comment on wowhead from 12/4/2006 (one day BEFORE the TBC patch was released)
http://db.vanillagaming.org/?quest=9210#wh-comments
"This quest is repeatable, but I have not done it a 2nd time yet so I don't know what reputation (if any) you get upon 2nd and further turn-ins."

**Comment by Lorilay2** [comment]
http://www.lurkerlounge.com/forums/printthread.php?tid=4055

**Comment by FenrirNost** [comment]
Shadow guard quest was repeatable as of June 2007: https://web.archive.org/web/20070604071106/http://wow.allakhazam.com:80/db/quest.html?wquest=9213
On the same page you can also see comments from Chirus and Kothal where Kothal mentions that you can apply 2 of the shadow guard enchants as of December 2, 2006

**Comment by strudelkuchen** [comment]
Sadly, all of your sources are post patch 1.11

**Comment by FenrirNost** [comment]
Yes. It is very sad that the only sources for this quest are from after it was added to the game. I too wish that we had historical records detailing events before they actually occurred. This would be proof that time travel IS possible and it would give me hope that one day I may not be a slave to the inexorable forward march of time.

Yes, a truly sad day indeed.

Oh, this bug report is for Elysium btw, which is on patch 1.11.
